components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/websocket/InboundWebsocketSourceHandler.java

```diff
@@ -169,6 +175,17 @@ private void handleHandshake(ChannelHandlerContext ctx, FullHttpRequest req) thr
         if (log.isDebugEnabled()) {
             WebsocketLogUtil.printHeaders(log, req, ctx);
         }
+
+        // This block is for the health check of the ports 8099 and 9099
+        if (!req.headers().contains(HttpHeaders.UPGRADE) && req.uri().equals("/health")) {
+            isHealthCheckCall = true;
+            FullHttpResponse httpResponse = new DefaultFullHttpResponse(HttpVersion.HTTP_1_1, HttpResponseStatus.OK);
+            httpResponse.headers().set("content-type", "text/plain; charset=UTF-8");
```

Comment:
Any reason for setting the content-type and content-length headers here? I do not see any content adding to the FullHttpResponse here. Maybe we can skip setting these headers unless content is supposed to be sent along with this response. WDYT?

Reply:
Removed the headers.